src/Time/StepControllers/SplitRemaining.hpp

```diff
@@ -24,6 +26,12 @@ namespace StepControllers {
 /// the slab.
 class SplitRemaining : public StepController {
  public:
+  /// \cond
```

Comment:
We should probably mark all the step controllers final since I don't expect anything will inherit from them. Correct me if I'm wrong :)

Reply:
Should we put something in our coding guidelines about the use of `final`?  If so, we should have a PR discussing the use of `final` and not hold up this PR

Reply:
Use final if the class shouldn't be used as a base class... There's not more to it than that.

Reply:
Then please put in a PR or file an issue to change our coding guidelines so everyone knows what to do in the future;  I have questions but want the discussion somewhere that can be found rather than buried in this PR.  We should also add what we decided about destructors in derived classes.